Makefile

```diff
@@ -118,41 +142,41 @@ deploy-pipe-hub-mce-sno:
 			-w name=ephemeral,emptyDir="" \
 			--timeout 5h \
 			--pod-template ./pipelines/resources/common/pod-template.yaml \
-			--use-param-defaults deploy-ztp-hub-mce  && \
-	tkn pr logs -L -n edgecluster-deployer -f
+			--use-param-defaults deploy-ztp-hub-mce  \
+			--showlog
```

Comment:
The `showlog` flag is mostly for CI? Just curious

Reply:
`--showlog` does the exact same thing as `tkn pr logs -L -n edgecluster-deployer -f` in this case but it's more reliable. Using the latter has the risk of showing the wrong execution logs if 2 pipelines are started in parallel. With the former we are sure of showing the logs for the right pipelinerun